src/js/edu-benefits/1990n/config/form.js

```diff
@@ -30,7 +32,94 @@ const formConfig = {
   },
   title: 'Apply for education benefits under the National Call to Service program',
   subTitle: 'Form 22-1990n',
-  chapters: {}
+  chapters: {
+    applicantInformation: {
+      title: 'Applicant Information',
+      pages: {
+        applicantInformation: applicantInformationPage(fullSchema1990n, {
+          fields: [
+            'veteranFullName',
+            'veteranSocialSecurityNumber',
+            'veteranDateOfBirth',
+            'gender',
+          ],
+          required: [
+            'veteranFullName',
+            'veteranDateOfBirth'
+          ],
+          isVeteran: true
+        }),
+        additionalBenefits: additionalBenefitsPage(fullSchema1990n, {
+          fields: [
+            'civilianBenefitsAssistance',
+            'civilianBenefitsSource',
+            'seniorRotcScholarshipProgram'
+          ]
+        }),
+        applicantService: {
+          title: 'Applicant service',
+          path: 'applicant/service',
+          initialData: {
+          },
+          uiSchema: {
+            'ui:title': 'Applicant service',
+            'view:applicantServed': {
+              'ui:title': 'Have you ever served on active duty in the armed services?',
+              'ui:widget': 'yesNo'
+            },
+            toursOfDuty: _.merge(toursOfDuty.uiSchema, {
+              'ui:options': {
+                expandUnder: 'view:applicantServed'
+              },
+              'ui:required': form => _.get('view:applicantServed', form)
+            }),
+            currentlyActiveDuty: {
+              'ui:options': {
+                expandUnder: 'view:applicantServed'
+              },
+              yes: {
```

Comment:
I don't love this property name but realize that's already defined in the backend :(